Drivers/NER/src/cascadiamc.c

```diff
@@ -0,0 +1,407 @@
+/**
+ * @file cascadiamc.c
+ * @author Hamza Iqbal
+ * @brief Source file for Motor Controller Driver
+ * @version 0.1
+ * @date 2023-08-22
+ * 
+ * @copyright Copyright (c) 2023
+ * 
+ */
+
+#include "cascadiamc.h"
+
+void cascadiamc_init(cascadiamc_t* mc)
+{
+
+}
+
+void cascadiamc_update(cascadiamc_t* mc, can_msg_t message)
```

Comment:
See above about callbacks, I guess we can keep this here for now but it should be moved when we write the MC monitor